pages/vendors.tsx

```diff
@@ -1,30 +1,32 @@
 import React, { useState, useEffect, useContext } from 'react';
 import Error from 'next/error';
-import { useMutation, useQuery } from '@apollo/react-hooks';
+import { useQuery } from '@apollo/react-hooks';
 import withGA from 'next-ga';
 import Router from 'next/router';
 
-import { GET_VENDORS_QUERY } from '../queries';
-
 import withData from '../hooks/withData';
+import Context from '../context';
+import { GET_VENDORS_QUERY } from '../queries';
+import { Vendor } from 'typings';
 
 import Heading from '../components/Heading';
 import Footer from '../components/Footer';
-
-import '../assets/styles/main.scss';
-import { Vendor } from 'typings';
 import Nav from '../components/Nav';
 import LoadingKeyboard from '../components/LoadingKeyboard';
 import VendorCard from '../components/VendorCard';
 import Meta from '../components/Meta';
-import Head from 'next/head';
-import Context from '../context';
 import CTACard from '../components/CTACard';
 
+import '../assets/styles/main.scss';
+import '../assets/styles/vendors.scss';
+import countryIsoList from '../assets/countries';
+import Head from 'next/head';
+
 interface VendorProps {}
 
 function Vendors(props: VendorProps): JSX.Element {
     const { loading, error, data } = useQuery(GET_VENDORS_QUERY);
+    const vendors = data.vendors;
```

Comment:
Deze kan beter naar 39. Als je een error krijgt is vendors undefined namelijk.

Reply:
Aight, twijfelde hier idd nog over